test/extensions/filters/http/ext_proc/ordering_test.cc

```diff
@@ -39,6 +39,9 @@ using testing::Unused;
 
 using namespace std::chrono_literals;
 
+// The buffer size for certain callbacks
```

Comment:
Seems like this is the decoder buffer limits? Maybe just call that out?

Reply:
Ok.